scene/main/window.cpp

```diff
@@ -972,6 +972,20 @@ DisplayServer::WindowID Window::get_window_id() const {
 	return window_id;
 }
 
+void Window::warp_mouse(const Vector2 &p_position) {
+	Transform2D xform = get_screen_transform();
+	Vector2 gpos = xform.xform(p_position);
+
+	if (transient_parent) {
```

Comment:
```suggestion
	if (transient_parent && !transient_parent->is_embedding_subwindows()) {
```

Or it will jump in the opposite direction when using single-window mode.

Reply:
Thanks!